packages/contracts-bedrock/src/L1/ETHLockbox.sol

```diff
@@ -71,9 +74,9 @@ contract ETHLockbox is ProxyAdminOwnedBase, Initializable, ISemver {
     mapping(address => bool) public authorizedLockboxes;
 
     /// @notice Semantic version.
-    /// @custom:semver 0.0.1
+    /// @custom:semver 1.0.0
     function version() public view virtual returns (string memory) {
-        return "0.0.1";
+        return "1.0.0";
```

Comment:
is this bump implying this is the last diff to this contract before production?

Reply:
Good point, it's hopefully the last bump although it depends if there are any audit fixes. What would you prefer?

Reply:
It would be nice to release at 1.0.0 but I don't feel strongly here. It's probably better to just release at 1.x.y than to forgot to come back and bump this later